tests/conf/datanode-test.toml.template

```diff
@@ -22,6 +22,12 @@ timeout_millis = 3000
 connect_timeout_millis = 5000
 tcp_nodelay = false
 
+[failure_detector]
```

Comment:
I'm not sure if we should add this section to `standalone.toml.template` and `metasrv.toml.template` also.

Reply:
I think this section shouldn't be added here. The `MetaSrvOptions` (which contains the `PhiAccrualFailureDetectorOptions`) is created in method `load_options` in `cmd/src/metasrv.rs`, which loads the config files provided by the user. So we can add this section in `config/metasrv.example.toml`. Also you can change the test `test_read_from_config_file` for this new section.